content/contributing.md

```diff
@@ -1,6 +1,6 @@
 +++
-title = "Getting Started KEDA"
-date = "2017-10-05"
+title = "Contributing to KEDA"
```

Comment:
This document is actually going to introduce new people to KEDA - What it is, how it could help, etc.

In terms of how to contribute, I would rather keep it out of this documentation page and just point to the contributing.md in our GitHub repo and keep them next to each other.

Reply:
Yep sounds good

Reply:
Thanks! Once that is changed it looks good to me!